messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/jms/JMSTopicControlHandler.java

```diff
@@ -143,11 +143,14 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
         final ServiceName serviceName = MessagingServices.getActiveMQServiceName(PathAddress.pathAddress(operation.get(ModelDescriptionConstants.OP_ADDR)));
 
         final String operationName = operation.require(ModelDescriptionConstants.OP).asString();
-        final String topicName = PathAddress.pathAddress(operation.require(ModelDescriptionConstants.OP_ADDR)).getLastElement().getValue();
+        String topicName = PathAddress.pathAddress(operation.require(ModelDescriptionConstants.OP_ADDR)).getLastElement().getValue();
```

Comment:
Please use OperationContext.getCurrentAddressValue();

Reply:
Of course :)